src/main/js/App.js

```diff
@@ -542,9 +542,19 @@ const App = () => {
                     </div>
                 </div>
                 <div>
-                    <div>{"Console"}</div>
-                    <textarea readOnly value={consoleText} class="console"></textarea>
```

Comment:
This used to have a class, we should add `className="console"` to the redone component.